src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/index.js

```diff
@@ -18,6 +18,7 @@ uiRoutes.when('/management/kibana/index', {
         indexPatterns: $injector.get('indexPatterns'),
         savedObjectsClient: $injector.get('Private')(SavedObjectsClientProvider),
         kbnUrl: $injector.get('kbnUrl'),
+        scopeApply: () => $scope.$apply(),
```

Comment:
While this gets the job done, it also pollutes the React component with Angular idioms like scope. Instead consider passing in a framework-agnostic property like `changeKbnUrl` which would be defined in this module here like this:

```
function changeKbnUrl(url) {
  scope.$evalAsync(() => kbnUrl.changePath(url));
}
```